sdk/keyvault/azure-keyvault-certificates/azure/keyvault/certificates/models.py

```diff
@@ -515,11 +515,24 @@ def request_id(self):
 class CertificatePolicy(object):
     """Management policy for a certificate.
 
-    :param attributes: the certificate attributes.
+    :param attributes: The certificate attributes. These are set by the service.
     :type attributes: ~azure.keyvault.certificates.models.CertificateAttributes
     :param str cert_policy_id: The certificate id.
-    :param key_properties: Properties of the key backing the certificate.
-    :type key_properties: ~azure.keyvault.certificates.models.KeyProperties
+    :param bool exportable: Indicates if the private key can be exported.
+    :param key_type: The type of key pair to be used for the certificate.
+        Possible values include: 'EC', 'EC-HSM', 'RSA', 'RSA-HSM', 'oct'
+    :type key_type: str or ~azure.keyvault.certificates.enums.KeyType
+    :param int key_size: The key size in bits. For example: 2048, 3072, or 4096
+        for RSA.
+    :param bool reuse_key: Indicates if the same key pair will be used on certificate
+        renewal.
+    :param curve: Elliptic curve name. For valid values, see KeyCurveName.
+        Possible values include: 'P-256', 'P-384', 'P-521', 'P-256K'
```

Comment:
Similarly, we have an enum for these, and a link to it doesn't become out of date.

Reply:
got rid of links to enums in comments, thanks!